### PR TITLE
Ash drake shapeshifting now converts health, fixes 60567

### LINKED ISSUE #60567: ash drake shapeshifting does not convert health when transforming back
#41638

### PR DESCRIPTION
**Pull request by jughu**
Miners shouldn't have 2 healthbars considering they are also super stocked on armor fixes #60567

:cl: Improvedname
fix: Ash drake shapeshift now converts your health like shapeshifting is supposed to
/:cl:

### CODE REVIEW

File: code/modules/spells/spell_types/shapeshift.dm [hunk 1]
```diff
@@ -136,7 +136,7 @@
 	name = "Dragon Form"
 	desc = "Take on the shape a lesser ash drake."
 	invocation = "RAAAAAAAAWR!"
-	convert_damage = FALSE
+	convert_damage = TRUE
```
Comment (Fikou) [inline code/modules/spells/spell_types/shapeshift.dm:139]: wont this be inherited from parent? you probably dont need the line at all

**Comment by Fikou** [comment]
this is intended even if its stupid

File: code/modules/spells/spell_types/shapeshift.dm (hunk 1 above)
Comment (jughu) [reply code/modules/spells/spell_types/shapeshift.dm:139]: Yeah fair

Comment (Watermelon914) [reply code/modules/spells/spell_types/shapeshift.dm:139]: make the change pls

**Comment by Ryll-Ryll** [comment]
eh, makes sense to me i guess

**Pull request merged**